src/styles/footer.scss

```
@@ -0,0 +1,19 @@
+.footer {
+  height: 200px;
+  background-image: url(../images/footer.png);
+  background-position: center;
+  background-size: cover;
+  z-index: -1;
```

Almost there, but the checklist item asks for the footer to be sticky at the bottom with z-index: -1.  You’ve already added the `z-index: -1`, but you also need to add the `position` and `bottom` properties. (Remember to add the background-color to `.main`. I left a comment there. 

```suggestion
  z-index: -1;
  position: sticky;
  bottom: 0;
```